packages/ckeditor5-bookmark/src/bookmarkediting.ts

```diff
@@ -39,4 +52,64 @@ export default class BookmarkEditing extends Plugin {
 			disallowAttributes: 'linkHref'
 		} );
 	}
+
+	/**
+	 * Defines the converters for the bookmark feature.
+	 */
+	private _defineConverters() {
+		const { editor } = this;
+		const { conversion, t } = editor;
+
+		conversion.for( 'dataDowncast' ).elementToElement( {
+			model: 'bookmark',
+			view: ( modelElement, { writer } ) => {
+				const emptyElement = writer.createEmptyElement( 'a', {
+					'id': modelElement.getAttribute( 'bookmarkId' )
+				} );
+
+				emptyElement.getFillerOffset = () => null;
```

Comment:
Why we need this?

Reply:
Good catch, it's also not needed here because `emptyElemnt` returns already `null` in this function :point\_right: https://github.com/ckeditor/ckeditor5/blob/b46c671710bd7402a70ba62a9ea10c2004b9559f/packages/ckeditor5-engine/src/view/emptyelement.ts#L91-L93